packages/loot-core/src/client/actions/modals.ts

```diff
@@ -15,12 +15,15 @@ export function pushModal<M extends keyof ModalWithOptions>(
   options: ModalWithOptions[M],
 ): PushModalAction;
 export function pushModal(name: OptionlessModal): PushModalAction;
+export function pushModal<M extends ModalType>(
+  name: M,
+  options?: FinanceModals[M],
+): PushModalAction;
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏nice!